katello/hooks/post/31-cdn_setting.rb

```diff
@@ -0,0 +1,10 @@
+if module_enabled?('katello')
+  param = get_custom_config('cdn_ssl_version')
+  if param
+    logger.info 'cdn_ssl_version param found, migrating to a Katello setting'
+    execute("foreman-rake -- config -k cdn_ssl_version -v #{param}")
```

Comment:
It's a good habit to always quote variables
```suggestion
    execute("foreman-rake -- config -k cdn_ssl_version -v '#{param}'")
```